### ISSUE TITLE
JDK8 Test failure: org.eclipse.jetty.io.SelectChannelEndPointTest.testIdle

### ISSUE THREAD

**Issue by joakime** [eclipse/jetty.project#2226]
`jetty-9.4.x` branch
Linux build.

```
Error Message
Expected: a value greater than <1000L>
     but: <991L> was less than <1000L>

Stacktrace
java.lang.AssertionError: 

Expected: a value greater than <1000L>
     but: <991L> was less than <1000L>
	at org.eclipse.jetty.io.SelectChannelEndPointTest.testIdle(SelectChannelEndPointTest.java:467)
```

**Comment by joakime** [comment]
Commit c751a8b1bbfdc73549bc9136f091b67270d9859e has been submitted to hopefully fix this.

**Comment by joakime** [comment]
Assigning to @gregw, as he's the author of the recent commit c751a8b1bbfdc73549bc9136f091b67270d9859e

**Comment by joakime** [comment]
The fix/commit for this, might also apply to Issue #2239

**Comment by joakime** [comment]
This is still triggering failures here and there.

https://jenkins.webtide.net/job/jetty.project/job/jetty-9.4.x/70/testReport/org.eclipse.jetty.io/SelectChannelEndPointTest/linux_jdk8___Test___jdk8___testIdle/history/

@sbordet & @gregw:  Since SelectChannelEndPoint is a deprecated class (replaced with SocketChannelEndPoint) can we just remove this SelectChannelEndPointTest (and associated SelectChannelEndPointSslTest)?

Perhaps, if needed, move some of the tests that are in SelectChannelEndPointTest to SocketChannelEndPointTest ?

**Comment by joakime** [comment]
These changes have been reviewed by @gregw and @sbordet, fixes have been pushed to `jetty-9.4.x`.  Closing.
